DataProcessing/EstimizeDataDownloader.cs

```diff
@@ -42,6 +41,8 @@ public abstract class EstimizeDataDownloader
             '_'
         };
 
+        protected readonly string FiscalYearQuarterByRelaeseId = "FiscalYearQuarterByRelaeseId.csv";
```

Comment:
I don't see the point in creating this file. If necessary, it should end up with the other files in  /data/alternative/estimize/FiscalYearQuarterByRelaeseId.csv.

Reply:
I'm trying to avoid reading and searching of fiscal year/quarter from different release files.